docs/configuration/listeners/ssl.rst

```diff
@@ -98,3 +99,12 @@ will be used.
 ecdh_curves
   *(optional, string)* If specified, the TLS connection will only support the specified ECDH curves.
   If not specified, the default curves (X25519, P-256) will be used.
+
+session_ticket_key_paths
+  *(optional, array)* Paths to keyfiles for encrypting and decrypting TLS session tickets. The
+  first keyfile in the array contains the key to encrypt all new sessions created by this context.
+  All keys are candidates for decrypting received tickets. This allows for easy rotation of keys
+  by, for example, putting the new keyfile first, and the previous keyfile second
+
+  Each keyfile must contain exactly 80 bytes of cryptographically-secure random data.  For example,
+  the output of ``openssl rand 80``.
```

Comment:
Could you add a paragraph saying that:
1) This feature is a security nightmare and that the session ticket keys must be (a) protected at the same or higher level that private keys of the TLS certificates, (b) rotated frequently (hourly or daily). See: https://www.imperialviolet.org/2013/06/27/botchingpfs.html.
2) Session Tickets are supported by Envoy by default and this feature should be only used in setups where TLS traffic is distributed across multiple Envoy instances.

Reply:
Yes, I will add this.